dev-python/nuitka/nuitka-0.6.5.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_5 python3_6 python3_7)
```

Comment:
You _can_ use a cleaner form here, `python3_{5..7}` or `python3_{5,6,7}` but up to you.

Reply:
I do use this in some of my ebuilds.